packages/components/form/src/form.vue

```diff
@@ -37,7 +37,9 @@ const formClasses = computed(() => {
   const { labelPosition, inline } = props
   return [
     ns.b(),
-    ns.m(formSize.value),
+    // todo: in v2.2.0, we can remove default
+    // in fact, remove it doesn't affect the final style
+    ns.m(formSize.value || 'default'),
```

Comment:
Add `default` to prevent the form from changing, in fact it doesn't affect the style at all.
We can remote it when `2.2` or not.